### DISCUSSION TITLE
Could you add multilang?

### DISCUSSION THREAD

**Discussion by ZekeriyaAY** [jpanther/congo#53] [Ideas]
- [x] Multilang support
- [x] You can add a multilang key, such as a theme toggle key. 

And I will open a pull request for Turkish language support soon.

**Comment by jpanther** [comment]
Multilang is something that I'm thinking about adding. I don't have any experience with multilang in Hugo though so it hasn't been a high priority for me at the moment. It might be something I look at for version 2 of the theme. Happy to look at a PR for this feature though.

Turkish language support is also very welcome! Open a PR anytime.
